webserver-configs/nginx.conf

```diff
@@ -80,6 +80,12 @@ server {
     }
     ## End - Let's Encrypt
 
+    ## Begin - Prevent file content downloads from leaking
```

Comment:
As mentioned in my last comment, this should not be necessary on a properly configured server.

Reply:
> As mentioned in my last comment, this should not be necessary on a properly configured server.


The correct configuration you said is the default configuration? 
I recognize that you are talking about the correct configuration, this is not necessary, but the information in this file is indeed critical and important, so I think a few lines of code can be changed to annotation form. You can be prompted to use UserFrosting, this file needs to be done in case of some incorrect configuration.

Reply:
Can you provide an example URL where you are able to get to this file (with reference to localhost)?  If there is a way to get to it with default configuration it is a serious security issue that would need to be addressed.

On a side note, UserFrosting is not meant to be installed under /var/www/public_html (or similar) which is the default root of nginx/apache etc. Rather, it should be installed somewhere like /usr/local/userfrosting, /usr/share/nginx/userfrosting (as used as the example in the nginx.conf file) or /var/www/userfrosting and then a configuration for nginx should be made with a root/directory alias at /usr/local/userfrosting/public, /usr/share/nginx/userfrosting/public, or /var/www/userfrosting/public using the 3 previous example locations.

Reply:
> Can you provide an example URL where you are able to get to this file (with reference to localhost)? If there is a way to get to it with default configuration it may be a serious security issue that would need to be addressed.
> 
> On a side note, UserFrosting is not meant to be installed under /var/www/public_html (or similar) which is the default root of nginx/apache etc. Rather, it should be installed somewhere like /usr/local/userfrosting, /usr/share/nginx/userfrosting (as used as the example in the nginx.conf file) or /var/www/userfrosting and then a configuration for nginx should be made with a root/directory alias at /usr/local/userfrosting/public, /usr/share/nginx/userfrosting/public, or /var/www/userfrosting/public using the 3 previous example locations.

Sorry, JSON files should not be cached.